commands/commands.go

```diff
@@ -172,3 +204,14 @@ func Autocomplete(options ...string) []*discordgo.ApplicationCommandOptionChoice
 	}
 	return choices
 }
+
+func isHerder(ic *discordgo.InteractionCreate) bool {
+	var herder bool
+	for _, role := range ic.Member.Roles {
+		if role == HerderRoleID {
+			herder = true
+			break
+		}
+	}
+	return herder || ic.Member.User.ID == AdminUserID
```

Comment:
I don't think we should keep this around. Skippy is also a herder and this bot does nothing that skippy should be able to do and that herders can't. Similar for var here https://github.com/discord-gophers/dgobot/pull/10/files#diff-b27e1ad555e9569bdd0e937e0bc9c0e0f3e9846f7d8ada8806ac4e7688d24c76R18 and the command flag.